tests/unittests/Foundation/NSCharacterSetTests.m

```diff
@@ -219,10 +219,4 @@ void testCharacter(NSCharacterSet* charSet, NSString* testString) {
 
     set = [NSMutableCharacterSet characterSetWithRange:NSMakeRange(0, 32)];
     ASSERT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
-
-    set = [NSMutableCharacterSet characterSetWithBitmapRepresentation:nil];
-    ASSERT_TRUE([set isKindOfClass:[NSMutableCharacterSet class]]);
-
-    set = [NSMutableCharacterSet characterSetWithContentsOfFile:nil];

```

Comment:
These tests are entirely invalid. You're not allowed to call characterSetWithBitmapRepresentation:nil. The argument must be non-nil. If it's nil it causes a failure on OSX.